### ISSUE TITLE
typescript code coverage stops working after update to 4.4.1 and code-coverage 3.5.1

### ISSUE THREAD

**Issue by jdmairs** [cypress-io/code-coverage#210]
**Logs and screenshots**
Please provide debug logs by running Cypress from the terminal with `DEBUG=code-coverage` environment variable set, see the [Debugging](https://github.com/cypress-io/code-coverage#debugging) section of the README file.

**Versions**

- What is this plugin's version? 3.5.1
- What is Cypress version? 4.4.1
- What is your operating system? OS X Catalina 10.15.4
- What is the shell? zsh
- What is the Node version? 12.14.1
- What is the NPM version? 6.14.4
- How do you instrument your application? babel Istanbul plugin
- When running tests, if you open the web application in regular browser, and open DevTools, do you see `window.__coverage__` object? Can you paste a screenshot?
- Is there `.nyc_output` folder? Is there `.nyc_output/out.json` file. Is it empty? Can you paste at least part of it so we can see the keys and file paths?
- Do you have any custom NYC settings in `package.json` (`nyc` object) or in other [NYC config files](https://github.com/istanbuljs/nyc#configuration-files)
- Do you run Cypress tests in a Docker container?

**Describe the bug**
Everything worked great with cypress 4.4.0 and coverage 3.1.0.  
One commit that simply bumps those up-to 4.4.1 and 3.5.1 and we get the situation that you used to get when even though you instrumented NYC/istanbul would not pull the coverage out.

**Comment by bahmutov** [comment]
1. If the plugin worked before in version X, but stopped after upgrading to version Y, please try the [released versions](https://github.com/cypress-io/code-coverage/releases) between X and Y to see where the breaking change was.
2. You have not provided any debug logs, we cannot help in this case

**Comment by jdmairs** [comment]
I started seeing this issue with 4.4.0 and 3.1.0 so then I knew it was my configuration.  Previously I had my cypress folder in an e2e subfolder under root with its own tsconfig.json and package.json.  I did this from an example on how to get Jest and cypress to not have type conflicts over describe.  

I absorbed the cypress folder back up to root, fixed my custom commands in typescript, and now the coverage works again.
